lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1316,6 +1316,15 @@ def setup_pe_conf(host, hosts, opts={})
                   update_pe_conf(opts[:custom_answers])
                 end
               end
+
+              step "Adding agent profiles" do
```

Comment:
Ah, nope, you are right. I forgot that we were already adding the profiles in beaker-answers.